setup.py

```diff
@@ -44,7 +44,11 @@
         'mozilla_django_oidc',
     ],
     include_package_data=True,
-    install_requires=['Django>1.7', 'python-jose', 'requests'],
+    install_requires=[
+        'Django>1.7',
```

Comment:
You are right, we don't support ``1.7``. I guess this happened because we assumed that ``>1.7`` means ``>=1.8`` which is wrong.